mmedit/models/backbones/sr_backbones/cain_net.py

```diff
@@ -0,0 +1,390 @@
+                Tensor: Output tensor with shape (n, c, h, w).
+                Tensor: CA tensor with shape (n, c, 1, 1).
+            else:
+                Tensor: Output tensor with shape (n, c, h, w).
+        """
+
+        res = x
+        out, ca = self.body(x)
+
+        if self.down_scale:
+            res = self.down_conv(res)
+        out += res
+
+        if self.return_ca:
+            return out, ca
+        else:
+            return out
+
+
+class ResidualGroup(nn.Module):
+    """Residual Group, consisting of a stack of residual channel attention,
+        followed by a convolution.
+
+    Args:
+        block_layer (nn.Module): nn.module class for basic block.
+        num_block_layers (int): number of blocks.
+        mid_channels (int): Channel number of the intermediate features.
+        kernel_size (int): Kernel size of ResidualGroup.
+        reduction (int): Channel reduction of CA. Default: 16.
+        act (function): activate function. Default: nn.ReLU(True).
+        norm (None | function): Norm layer. If None, no norm layer.
+            Default: None.
+    """
+
+    def __init__(self,
+                 block_layer,
+                 num_block_layers,
+                 mid_channels,
+                 kernel_size,
+                 reduction,
+                 act,
+                 norm=False):
+        super().__init__()
+
+        self.body = make_layer(
+            block_layer,
+            num_block_layers,
+            mid_channels=mid_channels,
+            kernel_size=kernel_size,
+            reduction=reduction,
+            norm=norm,
+            act=act)
+        self.conv_after_body = ConvNormWithReflectionPad(
+            mid_channels, mid_channels, kernel_size, stride=1, norm=norm)
+
+    def forward(self, x):
+        """Forward function for ResidualGroup.
+
+        Args:
+            x (Tensor): Input tensor with shape (n, c, h, w).
+
+        Returns:
+            Tensor: Output tensor with shape (n, c, h, w).
+        """
+
+        y = self.body(x)
+        y = self.conv_after_body(y)
+        return x + y
+
+
+@BACKBONES.register_module()
+class CAINNet(nn.Module):
+    """CAIN network structure.
+
+    Paper: Channel Attention Is All You Need for Video Frame Interpolation.
+    Ref repo: https://github.com/myungsub/CAIN
+
+    Args:
+        in_channels (int): Channel number of inputs. Default: 3.
+        out_channels (int): Channel number of outputs. Default: 3.
+        kernel_size (int): Kernel size of CAINNet. Default: 3.
+        num_block_groups (int): Number of block groups. Default: 5.
+        num_block_layers (int): Number of blocks in a group. Default: 12.
+        depth (int): Down scale depth, scale = 2**depth. Default: 3.
+        reduction (int): Channel reduction of CA. Default: 16.
+        norm (None | function): Norm layer. If None, no norm layer.
+            Default: None.
+        act (function): activate function. Default: nn.LeakyReLU(0.2, True)).
```

Comment:
The default activation function is not the same as in other modules. Is it intentional?